File: CS/10_Conversion/ConvertUnhideSlidesToPdf.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Presentation;

namespace ConvertUnhiddenSlidesToPdf
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            //Create a PPT document
            Presentation presentation = new Presentation();

            //Load PPT file from disk
            presentation.LoadFromFile(@"..\..\..\..\..\..\Data\HideSlide1.pptx");

            //Convert the PPT unhidden slides to PDF file format 
            presentation.SaveToPdfOption.ContainHiddenSlides = false;
            string result = "ToPdf.pdf";
            presentation.SaveToFile(result, FileFormat.PDF);

            //View File
            DocumentViewer(result);
        }

        private static void DocumentViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}

File: VB/10_Conversion/ConvertUnhideSlidesToPdf.vb
Imports System.ComponentModel
Imports System.Text
Imports Spire.Presentation

Namespace ConvertUnhiddenSlidesToPdf
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()

		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			'Create a PPT document
			Dim presentation As New Presentation()

			'Load PPT file from disk
			presentation.LoadFromFile("..\..\..\..\..\..\Data\HideSlide1.pptx")

			'Convert the PPT unhidden slides to PDF file format 
			presentation.SaveToPdfOption.ContainHiddenSlides = False
			Dim result As String = "ToPdf.pdf"
			presentation.SaveToFile(result, FileFormat.PDF)

			'View File
			DocumentViewer(result)
		End Sub

		Private Shared Sub DocumentViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace